### PR TITLE
adds missing ecosysytem repos, changes some repos to their source ver…

### PR DESCRIPTION
**Pull request by gcharang**
…sions for Komodo Platform

### CODE REVIEW

File: data/ecosystems/k/komodo.toml [hunk 1]
```diff
@@ -109,9 +109,12 @@ url = "https://github.com/KomodoPlatform/hyperdex"
 url = "https://github.com/KomodoPlatform/insight-ui-komodo"
 
 [[repo]]
-url = "https://github.com/KomodoPlatform/komodo"
+url = "https://github.com/jl777/komodo"
```
[the remark this answers is not in the document]
Comment (gcharang) [reply data/ecosystems/k/komodo.toml]: done

File: data/ecosystems/k/komodo.toml [hunk 2]
```diff
@@ -137,7 +140,7 @@ url = "https://github.com/KomodoPlatform/KomodoOcean"
 url = "https://github.com/KomodoPlatform/KomodoPaperWalletGenerator"
 
 [[repo]]
-url = "https://github.com/KomodoPlatform/KomodoPlatform"
+url = "https://github.com/jl777/SuperNET"
```
[the remark this answers is not in the document]
Comment (gcharang) [reply data/ecosystems/k/komodo.toml]: the repo https://github.com/KomodoPlatform/KomodoPlatform has been renamed to https://github.com/KomodoPlatform/atomicDEX-API

**Comment by gcharang** [comment]
I have pushed the requested changes.

**Comment by jubos** [comment]
Thank you for the changes.  Merged!

**Pull request merged**